static/scss/components/banners.scss

```diff
@@ -260,3 +212,111 @@
         }
     }
 }
+
+// Banner for users to register their premium subdomain
+.subdomain-banner {
```

Comment:
Not feeling confident about this class name but also struggling to grasp the proper naming convention. 

@craigcook: Do you have any sage advice on naming stuff to keep this Protocol'esque?